### PR TITLE
Move comment reply form to end of replies

### PR DESCRIPTION
**Pull request by n-wach**
Because newer comments are now at the bottom, it makes sense for the reply form to appear where the comment will be.

Clicking "reply" opens the form at the bottom, and causes you to scroll and focus on the text area.

Closes: #307

### CODE REVIEW

File: src/main/webapp/view/discussion/discussion-area.js [hunk 1]
```diff
@@ -40,32 +39,64 @@ export default class DiscussionArea {
...
-    this.#currentRootCommentElements = [];
     this.#nearestComments = [];
   }
 
   /**
    * Initialize the discussion area by loading the current comments.
    */
   async initialize() {
-    await this.loadDiscussion();
+    await this.updateDiscussion();
```
Comment (billie-cat) [inline src/main/webapp/view/discussion/discussion-area.js:49]: How is this related to your PR? Do you need to sync?

Comment (n-wach) [reply src/main/webapp/view/discussion/discussion-area.js]: Just merged, should only have the new changes now
